lib/ansible/modules/network/checkpoint/checkpoint_access_rule.py

```diff
@@ -71,27 +71,6 @@
       - State of the access rule (present or absent). Defaults to present.
     type: str
     default: present
-  auto_publish_session:
```

Comment:
since this module will be deprecated, plz remove these modules as is meaning do not change any thing in modules written by Ricky

Reply:
@justjais 
 If I don't change Riki's modules, the tests will fail, because his modules based on some code in the modules_utils (`checkpoint`) which I deleted..